recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "1.9.1" %}
+{% set version = "1.9.3" %}
 {% set numpy = "1.19" %}  # [py<=39]
 {% set numpy = "1.21" %}  # [py>39]
```

Comment:
Should we use `numpy` **1.22** for `py==310` and `win`?  See https://github.com/scipy/scipy/blob/de80faf9d3480b9dbb9b888568b64499e0e70c19/pyproject.toml#L42

Reply:
Their reason is specific to what the wheels on pypi contain. I don't see this affecting us.